bot.py

```diff
@@ -72,26 +38,12 @@ def load_responses_from_files(file: Path) -> str:
             )
             return result
     except FileNotFoundError as file_not_found_err:
-        logging.error(file_not_found_err)
-
-
-FRIEND_RESPONSE = load_responses_from_files(FRIENDS_RESPONSE_FILE)
-FAMILIAR_RESPONSE = load_responses_from_files(FAMILIAR_RESPONSE_FILE)
-HR_RESPONSE = load_responses_from_files(HR_RESPONSE_FILE)
-
-HR_KEY_WORDS = load_responses_from_files(HR_KEY_WORDS_FILE)
-
-
-async def show_selected_users():
-    async for dialog in CLIENT.iter_dialogs():
-        if dialog.id in FRIENDS_IDS:
-            logging.info(f"Selected friends username: {dialog.name}; ID: {dialog.id}")
-        elif dialog.id in FAMILIAR_IDS:
-            logging.info(f"Selected familiar username: {dialog.name}; ID: {dialog.id}")
+        logging.error(f"Err while load file - {file_not_found_err}")
+        return ""
```

Comment:
Why'd you return an empty string upon an exception? I'd advise `return None`, more self-explanatory.